### PR TITLE
feat: [CDS-36176]: delegate selector support 

### PR DESCRIPTION
**Pull request by dhruvUpadhyay07**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

**Comment by dhruvUpadhyay07** [comment]
trigger cypress

**Comment by dhruvUpadhyay07** [comment]
trigger rebase

**Comment by dhruvUpadhyay07** [comment]
trigger cypress

File: src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.module.scss [hunk 1]
```diff
@@ -102,3 +102,9 @@
 .conditionalExecutionTitle {
   font-size: var(--form-input-font-size) !important;
 }
+
+:global {
```
Comment (kapilkumawat86) [inline src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.module.scss:106]: Never do this, put it under local selector

**Review by kapilkumawat86 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**